components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -286,57 +300,78 @@ export const PrebuildDetailPage: FC = () => {
                                         </div>
                                     )}
                                 </div>
-                                {(currentPrebuild?.status?.taskLogs.some((t) => t.logUrl) || logNotFound) && (
-                                    <Tabs
-                                        value={taskId}
-                                        onValueChange={setSelectedTaskId}
-                                        className="p-0 bg-pk-surface-primary"
+                                <Tabs
+                                    value={taskId ?? "empty-tab"}
+                                    onValueChange={setSelectedTaskId}
+                                    className="p-0 bg-pk-surface-primary"
+                                >
+                                    <TabsList className="overflow-x-auto max-w-full p-0 h-auto items-end">
+                                        {isImageBuild && (
+                                            <TabsTrigger
+                                                value="image-build"
+                                                key="image-build"
+                                                data-analytics={JSON.stringify({ dnt: true })}
+                                                className="mt-1 font-normal text-base pt-2 px-4 rounded-t-lg border border-pk-border-base border-b-0 border-l-0 data-[state=active]:bg-pk-surface-secondary data-[state=active]:z-10 data-[state=active]:relative last:mr-1"
+                                            >
+                                                Image Build
+                                            </TabsTrigger>
+                                        )}
+                                        {currentPrebuild?.status?.taskLogs
+                                            .filter((t) => t.logUrl)
+                                            .map((task) => (
+                                                <TabsTrigger
+                                                    value={task.taskId}
+                                                    key={task.taskId}
+                                                    data-analytics={JSON.stringify({ dnt: true })}
+                                                    className="mt-1 font-normal text-base pt-2 px-4 rounded-t-lg border border-pk-border-base border-b-0 border-l-0 data-[state=active]:bg-pk-surface-secondary data-[state=active]:z-10 data-[state=active]:relative last:mr-1"
+                                                    disabled={isImageBuild}
+                                                >
+                                                    {task.taskLabel}
+                                                </TabsTrigger>
+                                            ))}
+                                    </TabsList>
+                                    <TabsContent
```

Comment:
In a follow-up, this will become a collection of tabs instead of one dynamic one.